src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,610 @@
+| block_timestamp      | string           | Block date/time string.                                                                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | May be `null`. If `true`, the contract address is has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam, phishing attempts, or suspicious activities. |
+| transaction_hash     | string           | Transaction ID of the transfer.                                                                  |
+| transaction_index    | string           | Transaction index of the transfer.                                                               |
+| log_index            | string           | Log index of the transfer.                                                                       |
+| value                | string           | Tranfer value.                                                                                   |
+| transaction_type     | string           | Transation type. Possible values are `Single`.                                                   |
+| from_address         | string           | Address of previous owner which sent the token(s).                                               |
+| to_address           | string           | Address of new owner which received the token(s).                                                |
+| status               | string           | Transfer status. Will be either `Send` or `Receive`. When the `from_address` and `to_address` are the same (i.e. sending to yourself), this value will be `Receive`. |
+| verified             | integer          |                                                                                                  |
+| operator             | string           | May be `null`.                                                                                   |
+
+<DevComment>
+`block timestamp` is a date time string, though I would expect a unix timestamp
+`verified` has no description.
+What are the other possible values for `transaction_type`?
+What is `operator`?
+What does `verified` mean?
```

Comment:
the moralis support team doesn't seem to know. hah. I will try to answer by myself.
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/667cc1f7-c428-4db3-8a13-d93a1712d489)

Reply:
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/cd56c462-9bba-45ce-b5ed-172426ee2b2a)
seems like they didnt remove this field but renamed to `verified_collection`?
```json
    {
      "block_number": "47961267",
      "block_timestamp": "2023-09-25T09:04:16.000Z",
      "block_hash": "0x0b256b4de5e82a8635e8a1bc3149d62c8d1332b6ad3b473a5ffab55a2071c301",
      "transaction_hash": "0x22dd86996e164c015b761e9b58b573ab37bf553ebd3ff73fd68399a117088321",
      "transaction_index": 75,
      "log_index": 471,
      "value": "0",
      "contract_type": "ERC1155",
      "transaction_type": "Single",
      "token_address": "0xc06a6ab4c2b0ed14ed5692adf324a9a463259c43",
      "token_id": "0",
      "from_address": "0x27ad1f808c1ef82626277ae38998afa539565660",
      "from_address_label": null,
      "to_address": "0xf622a6c52c94b500542e2ae6bcad24c53bc5b6a2",
      "to_address_label": null,
      "amount": "1",
      "verified": 0,
      "operator": "0x27ad1f808c1ef82626277ae38998afa539565660",
      "possible_spam": true,
      "verified_collection": false
    },
    {
      "block_number": "44506622",
      "block_timestamp": "2023-06-30T06:48:00.000Z",
      "block_hash": "0xcc40b0c8814a132b335f6f2c7cad25d85870ce671173057ee61a522df7b3fd79",
      "transaction_hash": "0x87dae6d10a60ecbe13d4a851e3589b2e320de57478f324564ab895de0731b43c",
      "transaction_index": 49,
      "log_index": 257,
      "value": "0",
      "contract_type": "ERC1155",
      "transaction_type": "Single",
      "token_address": "0x48c75fbf0452fa8ff2928ddf46b0fe7629cca2ff",
      "token_id": "5",
      "from_address": "0xf622a6c52c94b500542e2ae6bcad24c53bc5b6a2",
      "from_address_label": null,
      "to_address": "0xd37abcb646f21ac9abf9a01ffdee48704f18ad11",
      "to_address_label": null,
      "amount": "1",
      "verified": 1,
      "operator": "0xf622a6c52c94b500542e2ae6bcad24c53bc5b6a2",
      "possible_spam": false,
      "verified_collection": true
    }
```    

I think we can just include description from reference page
https://docs.moralis.io/web3-data-api/evm/reference/get-wallet-nft-transfers

they added response fields info
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/f649d91b-7c62-4afd-9097-0af14d13f5f2)
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/880cd359-92f9-42a1-893f-f41958b6d65a)